libcodechecker/server/api/report_server.py

```diff
@@ -181,31 +182,8 @@ def get_diff_hashes_for_query(base_run_ids, base_line_hashes, new_run_ids,
                                           msg)
 
 
-def get_run_result_query_helper(session, run_ids, report_filter, cmp_data,
-                                diff_hashes, is_unique, is_count=False):
-    filter_expression = process_report_filter_v2(report_filter)
-
-    if is_unique:
-        q = session.query(Report.bug_id,
-                          Report.checker_id,
-                          Report.checker_message,
-                          Report.severity,
-                          Report.file_id,
-                          func.max(ReviewStatus.status).label('status')) \
-            .group_by(Report.bug_id,
-                      Report.checker_id,
-                      Report.checker_message,
-                      Report.severity,
-                      Report.file_id)
-    elif is_count:
-        q = session.query(Report)
-    else:
-        q = session.query(Report,
-                          File,
-                          ReviewStatus)
-
-    return filter_report_filter(q, filter_expression, run_ids, cmp_data,
-                                diff_hashes)
+def group_by_unique_fields(q):
```

Comment:
Maybe it is just me but I find the name of this confusing. Some documentation might help. So `bug_id` is the only unique field in report table??